api/src/main/java/org/pentaho/platform/api/genericfile/model/IGenericFileContentWrapper.java

```diff
@@ -0,0 +1,13 @@
+package org.pentaho.platform.api.genericfile.model;
+
+import java.io.InputStream;
+
+public interface IGenericFileContentWrapper {
```

Comment:
Missing doclets.

Reply:
adding, along with HV Copyright to all applicable places...